_gsocproposals/2019/proposal_ActsGpuRace.md

```diff
@@ -0,0 +1,72 @@
+---
+title: GPU Race for Particle Hunting
+layout: gsoc_proposal
+project: ACTS
+year: 2019
+organization: LAL
```

Comment:
Since Pierre's affiliation is CEA, I guess it should be also listed as a organization participating on this project.

```suggestion
organization: 
 - LAL
 - CEA
```

Reply:
Ok. Working on it.